packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -1797,6 +1797,13 @@ export default class BackupNg {
           const path = `${dir}/${file}`
           try {
             const metadata = JSON.parse(String(await handler.readFile(path)))
+            if (metadata.mode === 'full') {
+              metadata.size = await handler
+                .getSize(resolveRelativeFromFile(path, metadata.xva))
+                .catch(err => {
+                  log.warn(`_listVmBackups, getSize`, { err })
```

Comment:
Maybe it would help for debugging if we add the path (or does it do that already)?

Also, we should add a short timeout to avoid blocking when listing the backups.

Reply:
Normally, we already have the path with the error, e.g "file xxxxx not found".